Library/Formula/android-sdk.rb

```diff
@@ -90,6 +90,9 @@ def install
       decoded_file.write Base64.decode64(File.read("adb.bash"))
       bash_completion.install decoded_file
     end
+
+    # automatically install platform and build tools
+    system "echo y | bash #{bin}/android --verbose update sdk --no-ui --all --filter platform-tools,build-tools-23.0.1"
```

Comment:
Yeh, meant to hard code. We don't have `pipe_output` here and it's essentially doing the same thing (but doesn't display incremental output which makes debugging hard). Also, `yes |` doesn't work.